Source/Core/VideoBackends/Vulkan/VKTexture.cpp

```diff
@@ -0,0 +1,340 @@
+// Copyright 2009 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <stdexcept>
```

Comment:
`<algorithm>`, `<cstddef>`, `<cstring>`, and `"Common/CommonTypes.h"` need to be included.